controller/src/bin/xcvradm.rs

```diff
@@ -747,6 +770,23 @@ async fn address_transceivers(
 
 // Column width for printing data below.
 const WIDTH: usize = 4;
+const POWER_WIDTH: usize = 5;
+
+fn print_power_mode(modules: ModuleId, modes: Vec<(PowerMode, Option<bool>)>) {
+    println!("FPGA Port Power Software override");
```

Comment:
Nit: it's not obvious that `Software override` is one column, versus two.  Maybe two spaces between columns?